<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonSegundo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTerceiro">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Terceiro</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSegundo"/>
    <addaction name="actionTerceiro"/>
    <addaction name="separator"/>
    <addaction name="menuSair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="menuSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionSegundo">
   <property name="text">
    <string>Segundo</string>
   </property>
  </action>
  <action name="actionTerceiro">
   <property name="text">
    <string>Terceiro</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
